newIDE/app/src/ResourcesList/index.js

```diff
@@ -79,6 +83,38 @@ export default class ResourcesList extends React.Component<Props, State> {
     this.props.onDeleteResource(resource);
   };
 
+  _scanForNewResources = () => {
+    const resourcesList = this
+    const project = this.props.project
+    const resourcesManager = project.getResourcesManager()
+    console.log('Scanning the project folder for new resources...');
+    const projectPath = path.dirname(project.getProjectFile());
+
+    const getDirectories = (src, callback) => {
+      glob(src + '/**/*.png', callback);
+    };
+    getDirectories(projectPath, function (err, res) {
```

Comment:
You're using a `function` here instead of an arrow `function`. This means that `this` will be hijacked to reference to the `function` (I know it's weird, it's a residue from early versions of JavaScript). https://medium.com/@thejasonfile/es5-functions-vs-es6-fat-arrow-functions-864033baa1a looks like a good explanation.

In short: when writing ES6 (which is the case for newIDE), always always use arrow functions :)

Reply:
ah good point. I didn't know about that. Thank you for the link :)

Reply:
Fixed now :)
The ES5/ES6 styling note should be added to the README if it already isnt imo